packages/@o3r/testing/schematics/ng-add/fixture/index.ts

```diff
@@ -1,21 +1,20 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import { chain, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
 import { getProjectFromTree } from '@o3r/schematics';
+
 import * as ts from 'typescript';
 
 /**
  * Add fixture configuration
  *
  * @param options @see RuleFactory.options
- * @param rootPath @see RuleFactory.rootPath
  * @param options.projectName
  * @param options.testingFramework
- * @param _rootPath
+ * @param installJest should install jest
```

Comment:
The name is incorrect: It does not **install** jest and it is **only** jest as it is mainly accessor.
A more correct name would be `registerUnitTestFixturePaths` of something like that.

You may also think about the playwright registration ?

Reply:
I prefer handling Playwright in a separate PR.
Tracked via https://github.com/AmadeusITGroup/otter/issues/657